main.js

```diff
@@ -0,0 +1,60 @@
+async function getPlayersData() {
+    var data = await fetch('https://my.api.mockaroo.com/players.json?key=a2166ce0');
+    return await data.json()
+}
+
+function capitalize(string) {
```

Comment:
`string` to bardzo kiepska nazwa zmiennej bo nic nie oznacza oprócz tego że to tekst